modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -129,7 +129,7 @@ $ echo "    bootstrapOSImage=${BOOTSTRAP_OS_IMAGE}"
 ----
 platform:
   baremetal:
-    bootstrapOSImage: http://<BAREMETAL_IP>:8080/<RHCOS_QEMU_PATH>?sha256=<RHCOS_QEMU_UNCOMPRESSED_SHA256>
+    bootstrapOSImage: <BOOTSTRAP_OS_IMAGE>
```

Comment:
This isn't clear to me if this is supposed to be kept as is or replaced. Is this supposed to be the actual variable (like how $BOOTSTRAP_OS_IMAGE is used earlier in the procedure), or is it a replaceable value, where the user is supposed to replace the value of $BOOTSTRAP_OS_IMAGE into this YAML here?

If it's a replaceable value for the user to replace, then our normal format for that is lowercase (<bootstrap_os_image>), and I would recommend to add a <1> callout to it here to tell the user to replace <bootstrap_os_image> with the value of $BOOTSTRAP_OS_IMAGE.